types/browan/TBDW100/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Door & Window Sensor, TBDW100 Tabs",
+  "name": "Tabs Door & Window Sensor, TBDW100",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed